src/utils.js

```diff
@@ -598,14 +598,14 @@ nv.utils.sanitizeWidth = function(width, container) {
 Calculate the available height for a chart.
 */
 nv.utils.availableHeight = function(height, container, margin) {
-    return nv.utils.sanitizeHeight(height, container) - margin.top - margin.bottom;
+    return Math.max(0,nv.utils.sanitizeHeight(height, container) - margin.top - margin.bottom);
```

Comment:
Because some time sanitizeWidth return zero and with the margin, availableWidth could return a negative value giving js error. So if sanitizeWidth is zero then available width too.